### ISSUE TITLE
pre-trained model / weight of loss / WGAN-GP loss

### ISSUE THREAD

**Issue by t109598032** [giles200619/VI-GAN#3]
Hi, thank you to provide this code, do you have any pre-trained model? And how do you know how to adjust the weight of the loss? I could not find the supplementary material which paper mentioned. And when i set WGAN-GP to True, the "d_loss" will be minus, is this reasonable? Thank you very much!

**Comment by giles200619** [comment]
Hi, 
I did not keep the pre-trained weight for this project. 
I asked for the supplementary material from the authors and implemented this based on what they describe. However, in practice, I found out that kicking out the GAN loss gives better results. The default weights of the losses as in train.py gives the best performance for me.  

(Basically I adjust the weights such that all of them are balanced, but please reach out to the authors if you would like to know about their original implementation.)
